Makefile

```diff
@@ -41,47 +41,53 @@ selinux: qm.pp ##  - Compresses the QM SELinux policy file (qm.pp)
 	cp tmp/qm.pp .; rm -rf tmp
 
 .PHONY: codespell
-codespell: ##  - Runs codespell to check for spelling errors
+codespell: ##             - Runs codespell to check for spelling errors
 	@codespell -S tmp,.git -L te -w
 
-clean: ##  - Removes generated files and dirs
+clean: ##             - Removes generated files and dirs
 	rm -f *~  *.tc *.pp *.pp.bz2
 	rm -rf tmp *.tar.gz ${RPM_TOPDIR}
 
-man: qm.8.md ##  - Generates the QM man page
+man: qm.8.md ##             - Generates the QM man page
 	go-md2man --in qm.8.md --out qm.8
 
 .PHONY: dist
-dist: ##  - Creates the QM distribution package
+dist: ##             - Creates the QM distribution package
 	tar cvz --transform s/qm/qm-${VERSION}/ -f /tmp/v${VERSION}.tar.gz ../qm
 	mv /tmp/v${VERSION}.tar.gz ./rpm
 
 .PHONY: rpm
-rpm: clean dist ##  - Creates a local RPM package, useful for development
+rpm: clean dist ##             - Creates a local RPM package, useful for development
 	mkdir -p ${RPM_TOPDIR}/{RPMS,SRPMS,BUILD,SOURCES}
 	cp ./rpm/v${VERSION}.tar.gz ${RPM_TOPDIR}/SOURCES
 	rpmbuild -ba \
 		--define="enable_qm_dropin_img_tempdir 0" \
+		--define="enable_qm_mount_bind_tty7 0" \
 		--define="_topdir ${RPM_TOPDIR}" \
 		--define="version ${VERSION}" \
 		${SPECFILE}
 
 # ostree target is a helper for everything required for ostree
 .PHONY: ostree
-ostree: qm_dropin_img_tempdir ##  - A helper for creating QM packages for ostree based distros
+ostree: qm_dropin_img_tempdir ##             - A helper for creating QM packages for ostree based distros
 
 .PHONY: qm_dropin_img_tempdir
-qm_dropin_img_tempdir: ## - Creates a QM RPM sub-package qm_dropin_img_tempdir
+qm_dropin_img_tempdir: ##            - Creates a QM RPM sub-package qm_dropin_img_tempdir
 	sed -i 's/%define enable_qm_dropin_img_tempdir 0/%define enable_qm_dropin_img_tempdir 1/' ${SPECFILE}
 	sed -i 's/^Version:.*/Version: ${VERSION}/' ${SPECFILE}
 	make VERSION=${VERSION} rpm
 
+.PHONY: qm_dropin_mount_bind_tty7
+qm_dropin_mount_bind_tty7: ##        - Creates a QM RPM sub-package to mount bind /dev/tty7 in the nested containers
+	sed -i 's/%define enable_qm_mount_bind_tty7 0/%define enable_qm_mount_bind_tty7 1/' ${SPECFILE}
+	sed -i 's/^Version:.*/Version: ${VERSION}/' ${SPECFILE}
+	make VERSION=${VERSION} rpm
```

Comment:
When used as a sub-make you probably should use `$(MAKE)` instead.
https://www.gnu.org/software/make/manual/html_node/Options_002fRecursion.html#Options_002fRecursion

Reply:
We do not use flags, but for security, probably also with `MAKEFLAGS=`.

Reply:
oooooops I forgot this comment.  @aesteve-rh could you please send a patch for this one?